net.biniou.LeBiniou.yml

```diff
@@ -0,0 +1,86 @@
+app-id: net.biniou.LeBiniou
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: lebiniou
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
When it supports wayland, this should become `fallback-x11`

Reply:
Ok, thanks.